### ISSUE TITLE
a11y: Not enough color contract in display of input schema

### ISSUE THREAD

**Issue by cjfreedman** [swagger-api/swagger-ui#9594]
### Q&A (please complete the following information)
 - OS: [e.g. macOS]  Windows 10
 - Browser: [e.g. chrome, safari]  Chrome
 - Version: [e.g. 22]  121
 - Method of installation: [e.g. npm, dist assets]  npm
 - Swagger-UI version: [e.g. 3.10.0]  Swagger UI React 5.11.3
 - Swagger/OpenAPI version: [e.g. Swagger 2.0, OpenAPI 3.0]  OpenAPI 3.0

### Content & configuration
Basic/ default

### Describe the bug you're encountering
We are using a tool called Sort Site to evaluate our site for 508 accessibility compliance.  It is flagging there is not enough color contrast in the display of the input schema: "Ensure that text and background colors have a 7:1 contrast ratio."
Violates WCAG 2.0 AAA 1.4.6 https://www.w3.org/WAI/WCAG21/Understanding/contrast-enhanced.html
This is occurring on the
- "td" (contained within the "tr class=property-row required")
- "span class=prop-type"
- "span class=prop-format"
- "span class=property primitive"
- "p" contained within the div with class "renderedMarkdown"

### To reproduce...
Open a POST method, Expand Schemas, expand an input schema, then example data elements.

### Expected behavior
enough color contrast.

### Screenshots
Here's the screenshot of the display
![image](https://github.com/swagger-api/swagger-ui/assets/86489380/acaf7ea2-373c-48d8-ad1d-a5a126de1451)

Here are the errors from Sort Site and the html context:
![image](https://github.com/swagger-api/swagger-ui/assets/86489380/85149c8c-4d63-4724-bd46-67f1d1ae75ce)
![image](https://github.com/swagger-api/swagger-ui/assets/86489380/cf27cc24-29c4-45c0-99fb-0ff5bcabb961)
![image](https://github.com/swagger-api/swagger-ui/assets/86489380/a9550f09-e7f4-4359-b75a-a02077c3218a)
![image](https://github.com/swagger-api/swagger-ui/assets/86489380/57bb741e-cd93-4996-98f7-454138ecd712)

### Additional context or thoughts
I would think the red asterisk marking "required" does not have enough color contrast either, because that element did not have enough contrast elsewhere.

**Comment by mgifford** [comment]
To find colors that meet the color contrast, I recommend https://contrast-finder.tanaguru.com/
